### PR TITLE
VH-38 Fix dates format and translations on vessel results

### PR DESCRIPTION
**Pull request by rdgfuentes**
- [x] Fix dates format on vessel search results
- [x] Translated vessel search results page

### CODE REVIEW

File: applications/vessel-history/public/locales/source/translations.json [hunk 1]
```diff
@@ -24,19 +24,23 @@
...
+    "callsign": "callsign",
+    "flag": "flag",
     "historyLabelByField": "{{label}} history for {{vesselName}}",
+    "imo": "imo",
+    "mmsi": "mmsi",
     "noAuthorizations": "No authorizations found",
     "plusPreviousValuesByField": "+{{quantity}} previous {{fieldLabel}}",
     "saveForOfflineView": "save vessel for offline view",
+    "source": "source",
+    "transmissionPlural": "transmissions",
```
Comment (fertandil87) [inline applications/vessel-history/public/locales/source/translations.json:36]: I think this is wrong.. https://www.i18next.com/translation-function/plurals

File: applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx [hunk 2]
```diff
@@ -37,43 +40,60 @@ const VesselListItem: React.FC<ListItemProps> = (props): React.ReactElement => {
...
+              ) : (
+                ' - '
+              )}
+            </Fragment>
+          ) : (
+            '-'
+          )}
         </div>
-        {props.saved && (
+        {/* {props.saved && (
```
Comment (fertandil87) [inline applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx:91]: the comment is ok?

**Review by fertandil87 — CHANGES_REQUESTED** [review]

**Review by fertandil87 — APPROVED** [review]

File: applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx (hunk 2 above)
Comment (rdgfuentes) [reply applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx:91]: Yes, that's not implemented yet

File: applications/vessel-history/public/locales/source/translations.json (hunk 1 above)
Comment (rdgfuentes) [reply applications/vessel-history/public/locales/source/translations.json]: Fixed

**Pull request merged**